shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/components/TabButton.kt

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.kotlinconf.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.jetbrains.kotlinconf.theme.grey50
+import org.jetbrains.kotlinconf.theme.greyWhite
+import org.jetbrains.kotlinconf.theme.text2
+import org.jetbrains.kotlinconf.theme.whiteGrey
+
+
+@Composable
+fun TabButton(tab: String, isSelected: Boolean, onSelect: () -> Unit) {
+    val background = if (isSelected)
+        MaterialTheme.colors.greyWhite
+    else
+        MaterialTheme.colors.whiteGrey
+
+    val textColor = if (isSelected) {
+        MaterialTheme.colors.whiteGrey
+    } else {
+        grey50
+    }
+
+    Box(
+        modifier = Modifier
+            .padding(top = 16.dp, bottom = 16.dp)
+            .background(
+                background,
+                shape = RoundedCornerShape(4.dp),
+            )
+            .clickable { onSelect() }
+    ) {
+        Text(
+            text = tab,
+            modifier = Modifier
+                .padding(start = 8.dp, end = 8.dp, top = 2.dp, bottom = 2.dp),
```

Comment:
there are vertical and horizontal paddings

Reply:
good catch, fixed